### PR TITLE
feat: Introduce small variant in Dropzone

### PR DESCRIPTION
**Pull request by diogoredin**
# Description

This PR introduces a smaller variant of the Dropzone.

## Changes

- [x] This has been done
- [ ] I still need to to this

## How to test

- Checkout this branch
- `$ yarn dev`

## Screenshots

Screenshot 2022-10-19 at 09 14 21

## To be notified

@TicketSwap/frontend 

## Links

[HG-2168](https://ticketswap.atlassian.net/browse/HG-2168)

### CODE REVIEW

File: src/components/Dropzone/index.stories.tsx [hunk 1]
```diff
@@ -29,6 +30,22 @@ export const Default = () => (
...
+  <>
+    <Dropzone
+      title="Upload PDF or Apple Wallet tickets"
+      subtitle="If your original file contains multiple tickets make sure to upload all of them, and we will let you select the ones you want to sell."
+      action="Drop files here or click here to select"
+      mobileAction="Select a file"
+      dropTitle="Release"
+      variant={DropzoneVariant.small}
+      onFileChange={file => console.log(file)}
+    />
```
Comment (viktorrenkema) [inline src/components/Dropzone/index.stories.tsx:43]: Only just now thought about this, but do we not add translation in Solar?

File: package.json [hunk 2]
```diff
@@ -54,7 +54,7 @@
   "dependencies": {
     "@reach/accordion": "^0.17.0",
     "@reach/auto-id": "^0.17.0",
-    "@reach/dialog": "^0.18.0",
+    "@reach/dialog": "^0.17.0",
```
Comment (viktorrenkema) [inline package.json:57]: Did we need to downgrade this dependency for this change?

**Review by viktorrenkema — APPROVED** [review]
One question, but lgtm otherwise!

File: src/components/Dropzone/index.stories.tsx (hunk 1 above)
Comment (viktorrenkema) [reply src/components/Dropzone/index.stories.tsx:43]: Ah nvm, this is the story haha

File: package.json (hunk 2 above)
Comment (diogoredin) [reply package.json:57]: This recent upgrade broke Storybook, will create an issue to check it out!

**Pull request merged**